06-aks-cluster.md

```diff
@@ -92,21 +94,25 @@ Now that the [cluster prequisites and shared Azure service instances are provisi
         > azuredeploy.parameters.region2.json
         ```
 
-    1.  Push the changes to your forked repo.
+    1.  Customize Flux to watch your own repo.
 
-        > :book: The DevOps team wants to automate their infrastructure deployments. In this case, they decided to use GitHub Actions. They are going to create a workflow for every AKS cluster instance they have to take care of.
+        > :book: GitOps allows a team to author Kubernetes manifest files, persist them in their git repo, and have them automatically applied to their clusters as changes occur. This reference implementation is for a multi cluster infrastructure, so Flux is going to use Kustomizer to deploy regions differenly by using a set of base manifest and patcing them when needed.
 
         ```bash
-        git add -A && git commit -m "setup GitHub CD workflow" && git push origin main
+        sed -i -e 's/<user-name>/${GITHUB_USER_NAME}/' cluster-manifests/base/cluster-baseline-settings/flux.yaml
```

Comment:
Is this the one that is now in `/cluster-baseline-settings/flux-system/flux.yaml`

Reply:
good catch! done | Addressed from [c2d7c](https://github.com/mspnp/aks-secure-baseline/pull/183/commits/c2d7c2557a5f4f7e57c7a75f1903362c4f0b03b0)